tools/ptrace.c

```diff
@@ -113,10 +124,27 @@ read_long(long *word, int pid, Elf64_Addr addr)
 int
 read_byte(char *byte, int pid, Elf64_Addr addr)
 {
+  /* Hold last ptrace_peekdata.  */
+  static char tlb_value[sizeof(tlb)];
```

Comment:
The size of this value should use sizeof(long), which is what ptrace returns.  It might or might not be the same size as ElfW(Addr), so better be precise.